mcs/class/corlib/System/AppDomain.cs

```diff
@@ -1000,11 +1000,17 @@ internal static String GetProcessGuid ()
 		}
 
 #if MONO_FEATURE_MULTIPLE_APPDOMAINS
+#if MONODROID
+		[Obsolete ("AppDomain.CreateDomain will no longer be supported in .NET 5 and later.  Consider AssemblyLoadContext when it becomes available https://docs.microsoft.com/en-us/dotnet/api/system.runtime.loader.assemblyloadcontext?view=netcore-3.0")]
```

Comment:
I wonder whether https://docs.microsoft.com/dotnet/core/dependency-loading/understanding-assemblyloadcontext is a better target page

Reply:
Agreed

Reply:
Shouldn't this be an aka.ms link?